experiments/tx_antenna_pattern_test.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
```

Comment:
should this file be in the test archive directory? Or was this created to run on the radar as a practical runtime test

Reply:
I had made it as an experiment for testing the actual operation of the radar. I think it can be archived.